### DISCUSSION TITLE
希望增加对其他分辨率的支持

### DISCUSSION THREAD

**Discussion by sa-ka-na114514** [moesnow/March7thAssistant#108] [General]
家里有台老电脑希望利用一下每天清日常，自己的电脑就好做其他事，但是老电脑最高只有1680×1050分辨率，如果大佬们能支持一下就好了，但是，谢谢大佬们了如果工作量很大那还是算了吧

**Comment by Elec4d0** [comment]
> 家里有台老电脑希望利用一下每天清日常，自己的电脑就好做其他事，但是老电脑最高只有1680×1050分辨率，如果大佬们能支持一下就好了，但是，谢谢大佬们了如果工作量很大那还是算了吧

可以用5块钱的hdmi诱骗器 + moonlight解决
